docs/whatsnew/5.2.rst

```diff
@@ -12,7 +12,17 @@ the 5.1 release.
 
 In particular, this release includes:
 
-* nothing yet
+
+.. _whatsnew-quantity-dtype:
+
+``Quantity`` data types
+=======================
+
+The default dtype argument for ``Quantity`` has been changed from `None`
+to ``numpy.floating``. With this dtype, Quantity will still upcast integers
```

Comment:
I wonder if we should be explicit to what we changed it. At some level, this is just a placeholder anyway. Could just be "... has been changed, so that one can now explicitly given ``dtype=None`` to get the same behaviour as numpy. Without an explicit argument, any integer values are still upcast to float, since that makes more sense for physical quantities."

Reply:
Agreed that it's just a placeholder. If someone want to know the specific value they can introspect `Quantity`.